docs/install/manage-add-ons-packages.md

```diff
@@ -0,0 +1,407 @@
+```
+
+In {file}`mx.ini`, specify the information to check out the add-on:
+
+```ini
+[collective.easyform]
+url=git@github.com:collective/collective.easyform.git
+branch=dev-branch-name
+extras=test
+```
+
+Add it to {file}`instance.yml` to let Zope know that this add-on should be loaded:
+
+```yaml
+default_context:
+    load_zcml:
+        package_includes: [
+            'project_title',
+            'collective.easyform',
+        ]
+```
+
+Stop the backend with {kbd}`ctrl-c`.
+Then apply your changes and start the backend.
+
+```shell
+make build-backend
+make start-backend
+```
+
+In your web browser, and assuming you are currently logged in as `admin`, visit the URL http://localhost:8080/Plone/prefs_install_products_form.
+An upgrade step might need to be performed in the Plone control panel.
+Follow the upgrade information, if present.
+Else click the {guilabel}`Install` button to complete installation of the add-on.
+
+
+(manage-pin-the-version-of-a-plone-package-against-constraints-label)=
+
+### Pin the version of a Plone package against constraints
+
+A version can **not** be pinned in `constraints.txt` when the package is mentioned in the constraints of Plone.
+Any other package version could be pinned in `constraints.txt`.
+ 
+Pin the version of a Plone package in {file}`mx.ini`:
+
+```ini
+[settings]
+# constraints of Plone packages
+version-overrides =
+    plone.api>=2.0.0a3
+```
+
+Apply your changes and restart backend:
+
+```shell
+make build-backend
+make start-backend
+```
+
+In your web browser, and assuming you are currently logged in as `admin`, visit the URL http://localhost:8080/Plone/prefs_install_products_form.
+An upgrade step might need to be performed in the Plone control panel.
+Follow the upgrade information, if present.
+
+
+(manage-checkout-a-plone-package-label)=
+
+### Check out a Plone package
+
+This section covers how to check out a Plone Core package for development.
+
+Add the Plone package you want to check out in {file}`mx.ini`.
+
+```ini
+[plone.restapi]
+url = git@github.com:plone/plone.restapi.git
+branch = master
+extras = test
+```
+
+Stop the backend with {kbd}`ctrl-c`.
+Then apply your changes and start the backend.
+
+```shell
+make build-backend
+make start-backend
+```
+
+In your web browser, and assuming you are currently logged in as `admin`, visit the URL http://localhost:8080/Plone/prefs_install_products_form.
+An upgrade step might need to be performed in the Plone control panel.
+Follow the upgrade information, if present.
+
+
+(manage-build-and-start-your-instance-label)=
+
+### Build and start your instance
+
+Build and run the Plone backend.
+
+```shell
+make build-backend
+make start-backend
+```
+
+In a web browser, visit http://localhost:8080/ to see that Plone is running.
+
+Your instance is running in the foreground.
+
+
+(manage-installation-details-label)=
+
+## Backend installation details
+
+The `Makefile` at the root of you project invokes commands in `backend/Makefile`.
```

Comment:
```suggestion
The `Makefile` at the root of your project invokes commands in `backend/Makefile`.
```

Reply:
Fixed in d24df6a800972d0e83c58b89372a89034c73e190